src/parser/jobs/gnb/modules/Ammo.tsx

```diff
@@ -97,16 +97,16 @@ export class Ammo extends CoreGauge {
 		}))
 
 		this.checklist.add(new Rule({
-			name: 'Cartridge Usage',
+			name: <Trans id="gnb.ammo.usage.title">Cartridge Usage</Trans>,
```

Comment:
Tiny nit - `Usage` should be lowercase here so the capitalization is consistent with other checklist items.

Reply:
done in [97cb994](https://github.com/xivanalysis/xivanalysis/pull/1705/commits/97cb9947eab0ff54b7af2e745a9c0c825f586252)

Reply:
just figured out I was dumb and forgot to `yarn extract`

Reply:
now it's done [1526650](https://github.com/xivanalysis/xivanalysis/pull/1705/commits/1526650dde71d79cdd5c0e37ea05fcd5aaa93fa3)